### PR TITLE
Add more links to CU from docs

### PR DESCRIPTION
**Pull request by jseldess**

### CODE REVIEW

File: cockroachcloud/frequently-asked-questions.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ This page answers the frequently asked questions about {{ site.data.products.ser
 
 {{ site.data.products.dedicated }} provides fully-managed, single-tenant CockroachDB clusters with no shared resources. {{ site.data.products.dedicated }} supports single and multi-region clusters in AWS and GCP.
 
+{{site.data.alerts.callout_success}}
```
Comment (ianjevans) [inline cockroachcloud/frequently-asked-questions.md:20]: This probably shouldn't be in the Dedicated FAQ, but in `serverless-faqs.md`.

File: v21.2/architecture/overview.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ CockroachDB was designed to create the source-available database we would want t
 
 However, you definitely do not need to understand the underlying architecture to use CockroachDB. These pages give serious users and database enthusiasts a high-level framework to explain what's happening under the hood.
 
+{{site.data.alerts.callout_success}}
```
Comment (ianjevans) [inline v21.2/architecture/overview.md:12]: We also should add a similar section to `cockroachcloud/architecture.md`.

Comment (jseldess) [reply v21.2/architecture/overview.md:12]: Done.

File: cockroachcloud/frequently-asked-questions.md (hunk 1 above)
Comment (jseldess) [reply cockroachcloud/frequently-asked-questions.md]: Done.

**Comment by jseldess** [comment]
@ianjevans, PTAL again. I added an "Online Courses" section at the end of the sidenav.

**Pull request merged**